Library/Formula/fuzzy-choose.rb

```diff
@@ -0,0 +1,17 @@
+class FuzzyChoose < Formula
+  desc "Hybrid string fuzzy matcher that uses both std{in,out} and a native GUI"
```

Comment:
Please use a shorter description. “`fuzzy-choose: <desc>`” should fit in 80 chars.